addons/pos_restaurant/models/pos_order.py

```diff
@@ -11,16 +11,16 @@ class PosOrderLine(models.Model):
     _inherit = 'pos.order.line'
 
     note = fields.Char('Internal Note added by the waiter.')
+    uuid = fields.Char(string='Uuid', readonly=True, copy=False)
     mp_skip = fields.Boolean('Skip line when sending ticket to kitchen printers.')
-    mp_dirty = fields.Boolean()
```

Comment:
why do you remove mp_dirty, but still using it in the frontend?

Reply:
because there's no need to store it anymore since the difference is always recomputed upon creating an Order object (when fetching or loading pos for example). So we know if something is dirty or not based on that